### PR TITLE
add front controller cleaning task

### PR DESCRIPTION
**Pull request by ihsanudin**
Make sure remove app_*.php after deployment

### CODE REVIEW

File: Mage/Task/BuiltIn/Symfony2/FrontControllerCleanTask.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    public function run()
+    {
+        $command = 'rm -rf web/app_*.php';
+        $result = $this->runCommandRemote($command);
+
+        return $result;
+    }
+}
```
Comment (eps90) [inline Mage/Task/BuiltIn/Symfony2/FrontControllerCleanTask.php]: PSR-2:
> All PHP files MUST end with a single blank line.

**Comment by eps90** [comment]
Just one comment added about checkstyle.
Please provide some examples of such configs in existing samples, it will be easier to document that later :)

File: Mage/Task/BuiltIn/Symfony2/FrontControllerCleanTask.php (hunk 1 above)
Comment (ihsanudin) [inline Mage/Task/BuiltIn/Symfony2/FrontControllerCleanTask.php]: Ok i will add a single blank line :-)

**Comment by ihsanudin** [comment]
This improvement do not require extra config on existing config files. Just make sure app_[stage].php is deleted.

**Comment by eps90** [comment]
Ok, but anyway, I'm gonna add this to the list of things to document in #213 . Thanks! ;)

**Pull request merged**